one-shot-device/README.md

```diff
@@ -1,73 +1,81 @@
+## Description
+
+One-Shot-Device is a microservice that allows the creator to register and authenticate devices for the CityXChange. It uses Integration Services and IOTA SSI.
```

Comment:
Please rephrase:
One-Shot-Device is a microservice that allows the creator to register and authenticate devices for the CityXChange project. It leverages the APIs of the Integration Services to create identities of SLUs and create channels where measured data will be tracked.

Reply:
Repharsed. Thanks!